### ISSUE TITLE
New release of ax-platform is raising `ModuleNotFoundError`

### ISSUE THREAD

**Issue by richardliaw** [facebook/Ax#387]
Is there a workaround for this?

```
(base) ➜  ~ git:(master) ✗ pip install -U ax-platform
Collecting ax-platform
  Downloading https://files.pythonhosted.org/packages/22/94/4664137378b701f1b5fe2612e2b111a20c05e88abaeeef9e44eee048ffa8/ax_platform-0.1.15-py3-none-any.whl (599kB)
     |████████████████████████████████| 604kB 3.0MB/s
Requirement already satisfied, skipping upgrade: pandas in ~/miniconda3/lib/python3.7/site-packages (from ax-platform) (1.0.5)
Requirement already satisfied, skipping upgrade: jinja2 in ~/miniconda3/lib/python3.7/site-packages (from ax-platform) (2.11.2)
Requirement already satisfied, skipping upgrade: plotly in ~/miniconda3/lib/python3.7/site-packages (from ax-platform) (4.9.0)
Requirement already satisfied, skipping upgrade: scipy in ~/miniconda3/lib/python3.7/site-packages (from ax-platform) (1.4.1)
Requirement already satisfied, skipping upgrade: botorch>=0.2.2 in ~/miniconda3/lib/python3.7/site-packages (from ax-platform) (0.3.0)
Requirement already satisfied, skipping upgrade: scikit-learn in ~/miniconda3/lib/python3.7/site-packages (from ax-platform) (0.22.2)
Requirement already satisfied, skipping upgrade: python-dateutil>=2.6.1 in ~/miniconda3/lib/python3.7/site-packages (from pandas->ax-platform) (2.8.1)
Requirement already satisfied, skipping upgrade: numpy>=1.13.3 in ~/miniconda3/lib/python3.7/site-packages (from pandas->ax-platform) (1.18.1)
Requirement already satisfied, skipping upgrade: pytz>=2017.2 in ~/miniconda3/lib/python3.7/site-packages (from pandas->ax-platform) (2020.1)
Requirement already satisfied, skipping upgrade: MarkupSafe>=0.23 in ~/miniconda3/lib/python3.7/site-packages (from jinja2->ax-platform) (1.1.1)
Requirement already satisfied, skipping upgrade: six in ~/miniconda3/lib/python3.7/site-packages (from plotly->ax-platform) (1.12.0)
Requirement already satisfied, skipping upgrade: retrying>=1.3.3 in ~/miniconda3/lib/python3.7/site-packages (from plotly->ax-platform) (1.3.3)
Requirement already satisfied, skipping upgrade: torch>=1.5 in ~/miniconda3/lib/python3.7/site-packages (from botorch>=0.2.2->ax-platform) (1.5.1)
Requirement already satisfied, skipping upgrade: gpytorch>=1.1.1 in ~/miniconda3/lib/python3.7/site-packages (from botorch>=0.2.2->ax-platform) (1.2.0)
Requirement already satisfied, skipping upgrade: joblib>=0.11 in ~/miniconda3/lib/python3.7/site-packages (from scikit-learn->ax-platform) (0.15.1)
Requirement already satisfied, skipping upgrade: future in ~/miniconda3/lib/python3.7/site-packages (from torch>=1.5->botorch>=0.2.2->ax-platform) (0.18.2)
Installing collected packages: ax-platform
  Found existing installation: ax-platform 0.1.14
    Uninstalling ax-platform-0.1.14:
      Successfully uninstalled ax-platform-0.1.14
Successfully installed ax-platform-0.1.15
(base) ➜  ~ git:(master) ✗ ipython
iPython 3.7.4 (default, Aug 13 2019, 15:17:50)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import ax
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-709cd9cacec7> in <module>
----> 1 import ax

~/miniconda3/lib/python3.7/site-packages/ax/__init__.py in <module>
     29     Trial,
     30 )
---> 31 from ax.modelbridge import Models
     32 from ax.service import OptimizationLoop, optimize
     33 from ax.storage import load, save

~/miniconda3/lib/python3.7/site-packages/ax/modelbridge/__init__.py in <module>
      8 from ax.modelbridge import transforms
      9 from ax.modelbridge.base import ModelBridge
---> 10 from ax.modelbridge.factory import (
     11     Models,
     12     get_factorial,

~/miniconda3/lib/python3.7/site-packages/ax/modelbridge/factory.py in <module>
     20 from ax.modelbridge.multi_objective_torch import MultiObjectiveTorchModelBridge
     21 from ax.modelbridge.random import RandomModelBridge
---> 22 from ax.modelbridge.registry import (
     23     Cont_X_trans,
     24     Models,

~/miniconda3/lib/python3.7/site-packages/ax/modelbridge/registry.py in <module>
     46 from ax.models.torch.botorch_kg import KnowledgeGradient
     47 from ax.models.torch.botorch_mes import MaxValueEntropySearch
---> 48 from ax.models.torch.botorch_modular.model import BoTorchModel
     49 from ax.models.torch.botorch_moo import MultiObjectiveBotorchModel
     50 from ax.utils.common.kwargs import (

ModuleNotFoundError: No module named 'ax.models.torch.botorch_modular'
```

**Comment by lena-kashtelyan** [comment]
Hi, @richardliaw, thanks so much for letting us know! I believe this is due to our forgetting to bump our BoTorch version requirement. If you upgrade `botorch` package to 0.3.1, this should go away. Let me know if it doesn't!

**Comment by richardliaw** [comment]
@lena-kashtelyan thanks for the fast response. I think it doesn't work actually:

```
Installing collected packages: torch, botorch
  Found existing installation: torch 1.5.1
    Uninstalling torch-1.5.1:
      Successfully uninstalled torch-1.5.1
  Found existing installation: botorch 0.3.0
    Uninstalling botorch-0.3.0:
      Successfully uninstalled botorch-0.3.0
Successfully installed botorch-0.3.1 torch-1.6.0

In [4]: exit
(base) ➜  ray git:(master) ✗
(base) ➜  ray git:(master) ✗ ipython
Python 3.7.4 (default, Aug 13 2019, 15:17:50)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import ax
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-709cd9cacec7> in <module>
----> 1 import ax

~/miniconda3/lib/python3.7/site-packages/ax/__init__.py in <module>
     29     Trial,
     30 )
---> 31 from ax.modelbridge import Models
     32 from ax.service import OptimizationLoop, optimize
     33 from ax.storage import load, save

~/miniconda3/lib/python3.7/site-packages/ax/modelbridge/__init__.py in <module>
      8 from ax.modelbridge import transforms
      9 from ax.modelbridge.base import ModelBridge
---> 10 from ax.modelbridge.factory import (
     11     Models,
     12     get_factorial,

~/miniconda3/lib/python3.7/site-packages/ax/modelbridge/factory.py in <module>
     20 from ax.modelbridge.multi_objective_torch import MultiObjectiveTorchModelBridge
     21 from ax.modelbridge.random import RandomModelBridge
---> 22 from ax.modelbridge.registry import (
     23     Cont_X_trans,
     24     Models,

~/miniconda3/lib/python3.7/site-packages/ax/modelbridge/registry.py in <module>
     46 from ax.models.torch.botorch_kg import KnowledgeGradient
     47 from ax.models.torch.botorch_mes import MaxValueEntropySearch
---> 48 from ax.models.torch.botorch_modular.model import BoTorchModel
     49 from ax.models.torch.botorch_moo import MultiObjectiveBotorchModel
     50 from ax.utils.common.kwargs import (

ModuleNotFoundError: No module named 'ax.models.torch.botorch_modular'
```

**Comment by lena-kashtelyan** [comment]
That's perplexing, let me investigate.

**Comment by stevemandala** [comment]
Update: Found the source of the problem, currently working on getting a fix committed (should be out in a bit) after which we will release a new version.

**Comment by stevemandala** [comment]
Fix is out 16cb8689, also released a new version (v0.1.16) which should be available from PyPl shortly

**Comment by richardliaw** [comment]
Thanks for the fast turnaround!
